ActindoWorkFlow/Integration/01_ConfigureQueueTypes.md

```diff
@@ -0,0 +1,307 @@
+
+### Procedure
+
+*Workflows > Workflows > Tab OVERVIEW > Select workflow > Select workflow version*
+
+![Workflow editor](/Assets/Screenshots/ActindoWorkFlow/Workflows/WorkflowEditor.png "[Workflow editor]")
+
+1. Select the action you want to assign a certain queue type to.      
+The action is highlighted and its settings are displayed in the settings side bar on the right.
+
+  ![Action](/Assets/Screenshots/ActindoWorkFlow/Workflows/Action.png "[Action]")
+
+2. Click the *Queue Type* drop-down list and select the desired queue type for the selected action. All available queue types are displayed in the list.     
+The selected queue type is displayed in the list.
+
+> [Info] The changes are automatically saved.
+
+[comment]:  <> (Is that right? Or do I have to click the deploy button to apply these changes?)
+
+### Next steps
+
+- [Delete a queue type](#delete-a-queue-type)
+- [Kill workers](#kill-workers)
+
+### See also
+
+- [User Interface workflows](/ActindoWorkFlow/UserInterface/00_UserInterface.md)
+- [Create a queue type](#create-a-queue-type)
+- [Edit a queue type](#edit-a-queue-type)
+- [Workflow and process elements](/ActindoWorkFlow/Overview/02_WorkflowProcessElements.md)
+- [Manage the workflows](/ActindoWorkFlow/Operation/01_ManageWorkflows.md)
+
+
+
+## Delete a queue type
+
+### Prerequisites
+
+At least one queue type is created, see [Create a queue type](#create-a-queue-type).
+
+### Procedure
+
+*Workflows > Queue Types > Tab OVERVIEW*
+
+![Overview](/Assets/Screenshots/ActindoWorkFlow/QueueTypes/Overview.png "[Overview]")
+
+1. Select the checkbox of the queue type you want to delete in the list of queue types.      
+The editing toolbar is displayed above the queue types list.
+
+  ![Toolbar](/Assets/Screenshots/ActindoWorkFlow/QueueTypes/Toolbar.png "[Toolbar]")
+
+2. Click the ![Delete](/Assets/Icons/Trash03.png "[Delete]") (Delete) button in the toolbar.   
+The queue type is deleted. The *QueueType has been deleted* pop-up window is displayed.
+
+  ![Queue type deleted](/Assets/Screenshots/ActindoWorkFlow/QueueTypes/QueueTypeDeleted.png "[Queue type deleted]")
+
+  > [Info] The *Default* queue type cannot be deleted.
+
+### Next steps
+
+- [Kill workers](#kill-workers)
+
+### See also
+
+- [User Interface workflows](/ActindoWorkFlow/UserInterface/00_UserInterface.md)
+- [Create a queue type](#create-a-queue-type)
+- [Edit a queue type](#edit-a-queue-type)
+- [Assign a queue type](#assign-a-queue-type)
+- [Workflow and process elements](/ActindoWorkFlow/Overview/02_WorkflowProcessElements.md)
+- [Manage the workflows](/ActindoWorkFlow/Operation/01_ManageWorkflows.md)
+
+
+
+
+## Kill workers
+
+Sometimes it may be necessary to stop certain process actions. Therefore, you have to kill the workers to avoid that these actions are executed. You can either kill the workers of a single queue type or kill all workers regardless their queue type.
+
```

Comment:
Info: The killed workers will be started again automatically unless you activate the "disable workflow workers" flag in the dev tools of the engine room